bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/BigtableOptions.java

```diff
@@ -44,6 +45,19 @@
       (int) TimeUnit.MILLISECONDS.convert(30, TimeUnit.MINUTES);
   public static final int BIGTABLE_ASYNC_MUTATOR_COUNT_DEFAULT = 2;
 
+  /**
+   * This describes the maximum size a bulk mutation RPC should be before sending it to the server
+   * and starting the next bulk call. Defaults to 1 MB.
+   */
+  public static final long BIGTABLE_MAX_BULK_RPC_SIZE_BYTES_DEFAULT = 1 << 20;
+
+  /**
+   * This describes the maximum number of individual mutation requests to bundle in a single bulk
+   * mutation RPC before sending it to the server and starting the next bulk call.
+   * The server has a maximum of 100,000.
+   */
+  public static final int BIGTABLE_MAX_BULK_REQUEST_COUNT_DEFAULT = 1_000;
```

Comment:
nit: I like the idea of prefix being meaningful for constants. BIGTABLE + BULK_MUTATE + ( MAX_MUTATION_COUNT | MAX_REQUEST_BYTES ) + DEFAULT

It is not clear why one thing is referred as RPC and the other as REQUEST